kbs/config/kubernetes/deploy-kbs.sh

```diff
@@ -22,18 +21,15 @@ kbs_cert="${k8s_cnf_dir}/base/kbs.pem"
     openssl pkey -in "${k8s_cnf_dir}/base/kbs.key" -pubout -out "${kbs_cert}"
 }
 
-if [ "${ARCH}" == "s390x" ]; then
-    if [ -n "${IBM_SE_CREDS_DIR:-}" ]; then
+if [ "$(uname -m)" == "s390x" ] && [ -n "${IBM_SE_CREDS_DIR:-}" ]; then
```

Comment:
It might be slightly nicer to split this conditional into two checks and warn people if they are on `s390x` but don't have `IBM_SE_CREDS_DIR` set.

Reply:
I'm not sure about this. Running on s390x without IBM_SE_CREDS_DIR is the point of this PR, so I'm not sure what we are warning against? I could add an info message to let people know the set-up?

Reply:
I've added an echo about ibm-se being used, I'm not sure if that helps?

Reply:
ah I see. well the echo can't hurt I guess.